textproc/bstring/Portfile

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                bstring
+github.setup        msteinert bstring 1.0.1 v
+github.tarball_from releases
+revision            0
+
+categories          textproc devel
+license             BSD
+maintainers         {@trodemaster icloud.com:manuals-unread2u} openmaintainer
+
+description         The Better String Library for C
+long_description    ${name} is a fork of Paul Hsieh's Better String Library. \
+                    It provides a comprehensive string library for C with \
+                    features like autotools build system, updated test suite \
+                    based on Check, Valgrind integration, and continuous \
+                    integration via GitHub Actions.
+
+homepage            https://github.com/msteinert/bstring
+
+checksums           rmd160  f967ae6d2ec3162419f8d8792fa094bd1eaa4acf \
+                    sha256  ba3d3726bf70a0920196199097bd7d341cf362eb7dbd64018a8936815817a0cc \
+                    size    488074
+
+depends_build       port:pkgconfig \
+                    port:autoconf \
+                    port:automake \
+                    port:libtool
+
+depends_lib         port:check
+
+use_autoreconf      yes
+
+configure.args      --enable-tests
+
+# Enable tests if check is available
+variant tests description "Build and run tests" {
+    depends_lib-append  port:check
+    configure.args-append --enable-tests
+}
```

Comment:
why both a variant and do the same thing in the "main" port, that doesn't seem to make much sense...

Reply:
I cleaned this up so tests variant does the tests and the main port doesn't